upgrades/upgrade_testing_framework/steps/step_expectation_selector.py

```diff
@@ -0,0 +1,16 @@
+from upgrade_testing_framework.core.framework_step import FrameworkStep
+
+class ExpectationSelector(FrameworkStep):
+    """
+    This step is where we collect applicable expectations based on our cluster details
+    """
+
+    def _run(self):
+        # Get the state we need
+
+        # Begin the step body
+        # Query the expectation knowledge base
+        self.logger.info("Querying applicable expectations...")
+
+        # Update our state
+        self.state.eligible_expectations = ['consistent-document-count']
```

Comment:
minor, but maybe worth adding a TODO-type comment that this is a temporary measure

Reply:
Please add the `eligible_expectations` to the FrameworkState as a declared member.  It seems like it will be here to stay, so it should be a first-class citizen, State-wise.

Reply:
Agreed for both will update in next revision